platforms/avr/drivers/spi_master.h

```diff
@@ -41,9 +41,18 @@ typedef int16_t spi_status_t;
 #ifdef __cplusplus
 extern "C" {
 #endif
+typedef struct spi_start_config_t {
+    pin_t    slavePin;
+    bool     lsbFirst;
+    uint8_t  mode;
+    uint16_t divisor;
+    bool     cs_active_low;
```

Comment:
rename to use `snake_case`?

Reply:
done in d9f3f018ec2dc986fde9ebc59a3edd269982a08d